messages/help.txt

```diff
@@ -1,7 +1,10 @@
-*ŠALÁT + MÄSO*
-stačí nahlásiť do ~11:00, že budete papať a dostanete šalátik + mäsko, aké si vypýtate z Billy. Bežne býva kuracie stehno, klobáska, no nájde sa občas aj sekaná, prsia, bôčik, ...
+*STÁLA PONUKA*
+Každý deň sa v kuchyni pripravujú pečené kuracie stehná (asi 3kg) a dusená zelenina.
```

Comment:
Well we could tell people to order the chicken if they want it and then prepare accordingly.